ci/jenkins/test-vmc.sh

```diff
@@ -457,22 +457,36 @@ function deliver_antrea {
     fi
 
     echo "====== Pulling old Antrea images ======"
```

Comment:
@XinShuYang @tnqn is this code used?
The "upgrade" test in this file is surprising to me, since we use an older Antrea image for some Nodes, but AFAIK we use a single manifest. So I guess we are testing "some new agents + some old agents + one new controller"?

Reply:
AFAIK `OLD_ANTREA_VERSION` is only used in matrix test, but as we discussed in #6161 this test has not been used for a long time.

Reply:
I was unaware of the code and the test. Version skew scenario can be covered by GIthub job test-compatible-N-1 and test-compatible-N-2. So it should be fine to remove this if it hasn't been used.

Reply:
Thanks. I will keep the changes I made even though they will be untested, and I assume this script will be updated correctly as part of #6161